cmd/gvproxy/main.go

```diff
@@ -285,6 +283,15 @@ func main() {
 	}
 }
 
+func getForwardsMap() map[string]string {
```

Comment:
This does not make a big difference, but I'd pass both `sshPort` and `sshHostPort` as args to this function, this limits the use of global variables.
Fine with me if you prefer to keep `getForwardsMap()` without args.

Reply:
Thanks, that makes sense. I changed the signature.